app/uk/gov/hmrc/helptosavetestadminfrontend/controllers/HelpToSaveApiController.scala

```diff
@@ -220,7 +220,7 @@ class HelpToSaveApiController @Inject()(http: WSHttp, authConnector: AuthConnect
                  |-H "Gov-Vendor-Version: ${params.govVendorVersion}" \\
                  |-H "Gov-Vendor-Instance-ID: ${params.govVendorInstanceId}" \\
                  |-H "Authorization: Bearer $token" \\
-                 | "${appConfig.apiUrl}/account"
+                 | "${appConfig.apiUrl}/${params.nino}/account"
```

Comment:
the NINO wouldn't appear in the URL for get account through the API platform (it appears in the get account endpoint in our BE)

Reply:
updated, thanks